test/test_result_files.py

```diff
@@ -0,0 +1,35 @@
+import difflib
+import sys
+import os
+
+test_file = "test/test.txt"
```

Comment:
These files names has been selected exclusively for debug purposes, I'll change them after Marta's PR merge